backend/src/tools/api.py

```
@@ -0,0 +1,515 @@
+)
+toolkit = RequestsToolkit(
+    requests_wrapper=TextRequestsWrapper(headers={}),
+    allow_dangerous_requests=True,
+)
+tools = toolkit.get_tools()
+system_message = """You have access to an API to help answer user queries.
+Here is documentation on the API:
+{api_spec}
+""".format(api_spec=api_spec)
+
+from langchain_community.utilities.requests import GenericRequestsWrapper
```

Ruff (0.11.9)

Redefinition of unused `GenericRequestsWrapper` from line 11

Remove definition: `GenericRequestsWrapper`

(F811)